Tests/CommandsTests/APIDiffTests.swift

```diff
@@ -43,63 +50,43 @@ class APIDiffTestCase: CommandsBuildProviderTestCase {
-        // It is kept around for abundance of caution. If "why is it needed"
-        // not identified by March 2025, then it is OK to remove it.
-        try XCTSkipIf(
-            Environment.current["SWIFTPM_TEST_API_DIFF_OUTPUT"] == "0",
-            "Env var SWIFTPM_TEST_API_DIFF_OUTPUT is set to skip the API diff tests."
-        )
-    }
-
-    func skipIfApiDigesterUnsupported() throws {
-      // swift-api-digester is required to run tests.
-      guard (try? UserToolchain.default.getSwiftAPIDigester()) != nil else {
-        throw XCTSkip("swift-api-digester unavailable")
-      }
-    }
-
-    func testInvokeAPIDiffDigester() async throws {
-        try skipIfApiDigesterUnsupported()
+    @Test(.requiresAPIDigester, arguments: [BuildSystemProvider.Kind.native, .swiftbuild])
+    func testInvokeAPIDiffDigester(buildSystem: BuildSystemProvider.Kind) async throws {
         try await fixture(name: "Miscellaneous/APIDiff/") { fixturePath in
             let packageRoot = fixturePath.appending("Foo")
             // Overwrite the existing decl.
             try localFileSystem.writeFileContents(
                 packageRoot.appending("Foo.swift"),
                 string: "public let foo = 42"
             )
-            await XCTAssertThrowsCommandExecutionError(try await execute(["diagnose-api-breaking-changes", "1.2.3"], packagePath: packageRoot)) { error in
-                XCTAssertFalse(error.stdout.isEmpty)
+            await expectThrowsCommandExecutionError(try await execute(["diagnose-api-breaking-changes", "1.2.3"], packagePath: packageRoot, buildSystem: buildSystem)) { error in
+                #expect(!error.stdout.isEmpty)
             }
         }
     }
 
-    func testSimpleAPIDiff() async throws {
-        try skipIfApiDigesterUnsupportedOrUnset()
+    @Test(.requiresAPIDigester, arguments: [BuildSystemProvider.Kind.native, .swiftbuild])
+    func testSimpleAPIDiff(buildSystem: BuildSystemProvider.Kind) async throws {
         try await fixture(name: "Miscellaneous/APIDiff/") { fixturePath in
             let packageRoot = fixturePath.appending("Foo")
             // Overwrite the existing decl.
             try localFileSystem.writeFileContents(
                 packageRoot.appending("Foo.swift"),
                 string: "public let foo = 42"
             )
-            await XCTAssertThrowsCommandExecutionError(try await execute(["diagnose-api-breaking-changes", "1.2.3"], packagePath: packageRoot)) { error in
-                XCTAssertMatch(error.stdout, .contains("1 breaking change detected in Foo"))
-                XCTAssertMatch(error.stdout, .contains("💔 API breakage: func foo() has been removed"))
+            await expectThrowsCommandExecutionError(try await execute(["diagnose-api-breaking-changes", "1.2.3"], packagePath: packageRoot, buildSystem: buildSystem)) { error in
+                expectMatch(error.stdout, .contains("1 breaking change detected in Foo"))
```

Comment:
suggestion: can we use `#expect(error.stdout.contains(".."))` instead of supporting a helper function?

This comment applies to all other instances.